nodeJS/express/environment_variables.md

```diff
@@ -0,0 +1,112 @@
+### Introduction
+
+When you run your code, your code runs in an individual environment. Run your code on a different machine? Different environment. Host your website somewhere like Netlify or Vercel? Different environment to your machine. Each environment is a unique collection of tools and configurations.
```

Comment:
I'm not sure if "individual environment" is the right phrasing here. Maybe "specific environment" or "defined environment".

Reply:
"specific" probably feels a bit smoother here :+1: